src/Components/Basket/Basket.css

```diff
@@ -0,0 +1,50 @@
+
+
+
```

Comment:
remove unnecessary lines. I think we have to configure eslint for project, to handle the same style of code in all files.